sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -2195,22 +2195,4 @@ class DDLParserSuite extends AnalysisTest {
       parsePlan("COMMENT ON TABLE a.b.c IS 'xYz'"),
       CommentOnTable(UnresolvedTable(Seq("a", "b", "c")), "xYz"))
   }
-
-  test("create table - without using") {
```

Comment:
after a second thought, can we keep the test but ignore it? We still need these tests after the syntax unification is done. We can add a comment to reference the JIRA ticket.

Reply:
same for other removed tests.

Reply:
Yes. +1 for @cloud-fan 's suggestion. (ignoring instead of removal).

Reply:
It's straightforward for removed tests - I'll do that.

What about the tests being reverted as Hive create table is not integrated with DSv2?
(I'm referring `sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala`) 
I guess they still need to be reverted instead of ignore.

Reply:
+1. Maybe ignoring tests and add TODO comments to them is better.

Reply:
It cannot be simply changed to "ignore" instead of "test" as we're removing the config. I'll comment out instead.

Reply:
So, just remove the conf after revert?

Reply:
Sorry but could you please elaborate? Removing config is what this patch proposed, and it's a part of revert.

Reply:
I think he meant to remove `withSQLConf(SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED.key -> "false")`. We don't need the config anyway, if we unify the syntax.

Reply:
Yeah I see. That's one of valid approaches and good idea. Maybe then we need to remove some tests which depend on the config (two tests for on and off).